modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -384,6 +384,30 @@
 		"issueKey": "LPS-194818",
 		"to": "addFileEntry(null, param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, null, param#6#, param#7#, param#8#, null, null, param#9#)"
 	},
+	{
+		"classNames": [
+			"PortletDisplay"
+		],
+		"from": "getPortletInstanceConfiguration(Class<T>)",
+		"issueKey": "LPS-195850",
+		"newImports": [
+			"com.liferay.portal.configuration.module.configuration.ConfigurationProviderUtil",
+			"com.liferay.portal.kernel.service.ServiceContextThreadLocal"
+		],
+		"to": "ConfigurationProviderUtil.getPortletInstanceConfiguration(param#0#, ServiceContextThreadLocal.getServiceContext().getThemeDisplay())"
+	},
+	{
+		"from": "portletDisplay.getPortletInstanceConfiguration(Class<T>)",
+		"issueKey": "LPS-195850",
+		"newImports": [
+			"com.liferay.portal.configuration.module.configuration.ConfigurationProviderUtil"
+		],
+		"skipParametersValidation": true,
```

Comment:
@MicaelleSilvaa, is this really necessary? Can you please share more details about it?

Thanks

Reply:
In catchAll, when we pass a parenthesis in the from it is necessary to validate the type of the parameter, however, in this case we are declaring a class as a parameter and in jsp cases we don't do the test by specifying the type of the parameter and creating test cases for error: (null, test123 and 123). It's not possible to validate. That's why skipParametersValidation is necessary.

Reply:
@MicaelleSilvaa, based on our off-line conversation from yesterday, the goal is try to implement the check without use the attribute `skipParametersValidation`. Can you please give it a try?

Thanks